src/v/config/node_config.cc

```diff
@@ -180,6 +180,15 @@ node_config::node_config() noexcept
       "injection",
       {.visibility = visibility::tunable},
       std::nullopt)
+  , verbose_logging_timeout_max(
+      *this,
+      "verbose_logging_timeout_max",
+      "Maximum duration in seconds for verbose (i.e. TRACE or DEBUG) logging. "
+      "Values configured above this will be clamped. If null there is no "
+      "limit, but defaults to 600s. Can be overridded in the Admin API on a "
+      "per-request basis.",
+      {.visibility = visibility::tunable},
+      std::chrono::seconds{600})
```

Comment:
suggestion: I think this should default to `std::nullopt` so today's behavior continues to be the default.  We will ensure our fmc offerings will set node configs will set this (600s seems reasonable).

Reply:
oh ya, i misread the ticket